examples/md-flexible/tests/ThermostatTest.cpp

```diff
@@ -111,13 +117,45 @@ TEST_F(ThermostatTest, MultiComponentTest) {
     EXPECT_NEAR(temperature, targetTemperature1, 0.1);
   }
 
+  // Check no particles have zero temperature (indicating brownian motion hasn't been applied).
```

Comment:
```suggestion
  // Check no particles have zero temperature (ensuring brownian motion has been applied).
```

Reply:
just rephrasing that because I was a bit confused, just a suggestion if it makes sense to you.